### PR TITLE
Update translator to use neon_lang_config

### PR DESCRIPTION
**Pull request by NeonDaniel**
Update translator to use neon_lang_config to match detector
Update translator defaults to valid neon plugins

### CODE REVIEW

**Review by JarbasAl — APPROVED** [review]

File: test/test_language.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+class LanguageTests(unittest.TestCase):
+    def test_get_lang_config(self):
+        config = get_lang_config()
+        self.assertIsInstance(config, dict)
+        self.assertIsInstance(config['internal'], str)
+        self.assertIsInstance(config['user'], str)
+
+    def test_get_language_dir_valid(self):
```
Comment (NeonKirill) [inline test/test_language.py:43]: I think we could predefine the dict structure of gotten/expected here, that will make it easier to add/remove language dirs as needed

**Review by NeonKirill — CHANGES_REQUESTED** [review]

File: test/test_language.py (hunk 1 above)
Comment (NeonDaniel) [reply test/test_language.py:43]: Good idea; I'll make that change

File: .github/workflows/core_tests.yml [hunk 2]
```diff
@@ -30,4 +30,7 @@ jobs:
         run: |
           pytest test/test_skill_utils.py
         env:
-          GITHUB_TOKEN: ${{secrets.neon_token}}
\ No newline at end of file
+          GITHUB_TOKEN: ${{secrets.neon_token}}
+      - name: Test Language
```
Comment (NeonKirill) [inline .github/workflows/core_tests.yml:34]: I would suggest generating xml artifacts on each test run to check if something goes wrong

**Review by NeonKirill — APPROVED** [review]
Minor suggestion, otherwise looks good!

File: .github/workflows/core_tests.yml (hunk 2 above)
Comment (NeonDaniel) [reply .github/workflows/core_tests.yml:34]: Opened a separate issue for this since it should apply to all tests

**Pull request merged**